integration-tests/static-buffer/lib.rs

```diff
@@ -75,5 +86,34 @@ pub mod static_buffer {
 
             Ok(())
         }
+
+        #[ink_e2e::test]
+        async fn buffer<Client: E2EBackend>(mut client: Client) -> E2EResult<()> {
+            // given
+            let mut constructor = StaticBufferRef::new_default();
+
+            // when
+            let contract = client
+                .instantiate("static_buffer", &ink_e2e::bob(), &mut constructor)
+                .submit()
+                .await
+                .expect("instantiate failed");
+            let call_builder = contract.call_builder::<StaticBuffer>();
+
+            // then
+            let get = call_builder.buffer();
+            let get_res = client.call(&ink_e2e::bob(), &get).submit().await?;
+
+            let debug_msg = get_res.debug_message();
+            let msgs: Vec<&str> = debug_msg.split('\n').collect();
+            let ptr1 = u64::from_str_radix(msgs[0].trim_start_matches("0x"), 16).unwrap();
+            let ptr2 = u64::from_str_radix(msgs[1].trim_start_matches("0x"), 16).unwrap();
+            let align = u64::from_str_radix(msgs[2], 10).unwrap();
+
```

Comment:
This seems a bit hacky to rely on the debug_message here. 
Can't we just do that in the ink! message itself and assert it does not panic, or return the values and assert here?